qiskit/transpiler/preset_passmanagers/__init__.py

```diff
@@ -25,6 +42,23 @@
    level_1_pass_manager
    level_2_pass_manager
    level_3_pass_manager
+
+.. _stage_generators:
+
+Stage Generator Functions
+-------------------------
+
+.. autosummary::
+   :toctree: ../stubs/
+
+   ~qiskit.transpiler.preset_passmanagers.common.generate_control_flow_options_check
+   ~qiskit.transpiler.preset_passmanagers.common.generate_error_on_control_flow
+   ~qiskit.transpiler.preset_passmanagers.common.generate_unroll_3q
+   ~qiskit.transpiler.preset_passmanagers.common.generate_embed_passmanager
+   ~qiskit.transpiler.preset_passmanagers.common.generate_routing_passmanager
+   ~qiskit.transpiler.preset_passmanagers.common.generate_pre_op_passmanager
+   ~qiskit.transpiler.preset_passmanagers.common.generate_translation_passmanager
+   ~qiskit.transpiler.preset_passmanagers.common.generate_scheduling
```

Comment:
This documentation is a bit misleading - the `currentmodule` is `qiskit.transpiler.preset_passmanagers`, but these functions aren't actually available from that namespace.  We either need to change the `module` for the documentation of these, or import them into the top-level namespace (my preferred choice - I like packages to properly define a public API, and not require users to go searching in the code for the module stuff happens to be defined it).

Reply:
I was on the fence about re-exporting it form the root here. I opted to not do it in this PR since I wanted to leave this as purely documentation changes. But I can add it if you think it's necessary here. Otherwise I think we can do this in a follow up PR. TBH, I'm not really sure why I didn't re-export these functions when I added these in #6403.